recipe/conda_build_config.yaml

```
@@ -56,7 +56,8 @@ cuda_compiler:
 cuda_compiler_version:
   - None
   - 11.8                       # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
-  - 12.0                       # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+  - 12.4                       # [linux and ppc64le and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
```

```suggestion
  - 12.4                       # [ppc64le and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
```
ppc64le implies Linux.

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocessing-selectors

---

It's better to be explicit